sr_unix/gtmci.c

```diff
@@ -204,21 +208,23 @@ int gtm_cij(const char *c_rtn_name, char **arg_blob, int count, int *arg_types,
 {
 	boolean_t		need_rtnobj_shm_free;
 	callin_entry_list	*entry;
+	int4			*lnr_tab_ent;
 	mstr			label, routine;
 	int			has_return, i, len;
 	rhdtyp          	*base_addr;
-	uint4			inp_mask, out_mask, mask;
+	char			*xfer_addr;
+	uint4			inp_mask, out_mask, mask, omask;
```

Comment:
Perhaps I don't understand your question. The omask variable is used in both functions in the migration of the parms out of varargs and into the parm_blk structure. gtm_cij() is the Java interface while gtm_ci_exec() deals with the gtm_ci() and gtm_cip() calls. While there is functional overlap (at a high level) between these routines, they are actually quite different. Can you otherwise clarify or restate your question?

Reply:
Below is a grep of omask in gtmci.c. I see it is being set and updated. But I don't see it ever USED. What am I missing?

> grep omask sr_unix/gtmci.c
        uint4                   inp_mask, out_mask, mask, omask;
        for (i = 0, mask = inp_mask, omask = out_mask;
             ++i, mask >>= 1, omask >>= 1, java_arg_type++, arg_blob_ptr += GTM64_ONLY(1) NON_GTM64_ONLY(2))
        uint4                   inp_mask, out_mask, mask, omask;
        for (i = 0, mask = ~inp_mask, omask = out_mask; i < entry->argcnt; ++i, mask >>= 1, omask >>= 1)

Reply:
Now I see what you mean. Looked at it too fast before. Artifacts of an earlier revision. Now removed.